ui/static/lib/p5/compat/data.js

```
@@ -0,0 +1,343 @@
+		return new p5.NumberDict(key, value);
+	};
+
+	p5.TypedDict = class TypedDict {
+		constructor(key, value) {
+			if (key instanceof Object) {
+				this.data = key;
+			} else {
+				this.data = {};
+				this.data[key] = value;
+			}
+			return this;
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)